integration-tests/manifests/sample-deployment.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
I would use the exact same resource spec including the same image for all 3, i.e. Deployment, Daemonset and StatefulSet. Just change the Kind and keep them as simple as we can.
We can just call them sample-deployment, sample-daemonset, sample-statefulset etc.

Reply:
agreed, they should all use the same sample "app". Which it seems like in your case I would just use nginx